docker/Dockerfile

```diff
@@ -36,6 +36,21 @@ RUN touch .dockerenv \
     wget \
     whois \
     xz \
+    # The following packages are necessary for building Das U-Boot and Mender
```

Comment:
I wonder if u-boot-tools is missed?

Reply:
You're right, but in Alpine it's called `uboot-tools`.